ChangeLog

```diff
@@ -69,6 +69,8 @@
  * SimplicialCubature (openturns.experimental)
 
 ==== API changes ====
+ * Deprecated MetaModelValidation(inputSample, outputSample, metaModel) is deprecated in favor of MetaModelValidation(outputSample, metamodelPredictions)
+ * Deprecated MetaModelValidation::computePredictivityFactor method, use computeR2Score
```

Comment:
move this to the 1.23 section, mention new validation classes

Reply:
Fixed at 53f39e6e5